scripts/split-e2e-tests-codebuild.ts

```diff
@@ -240,6 +253,9 @@ const splitTestsV3 = (
       if (j.useParentAccount) {
         tmp.env.variables.USE_PARENT_ACCOUNT = 1;
       }
+      if (j.disableCoverage) {
+        tmp.env.variables.DISABLE_COVERAGE = 1;
+      }
```

Comment:
can we rename `j` and `tmp` to something meaningful?

Reply:
updated